### DISCUSSION TITLE
v1.1.1

### DISCUSSION THREAD

**Discussion by dayland** [microsoft/PubSec-Info-Assistant#727] [Announcements]
# What's New
- Support for migrating or upgrading from v1.0 to v1.1.x. More details can be found in the documentation at [Moving from v1.0 to v1.1](https://github.com/microsoft/PubSec-Info-Assistant/blob/main/docs/deployment/move_or_migrate.md)
- API Version updates
   - openai=1.17.0
   - langchain=0.1.16
   - langchain-openai=0.1.3
   - tiktoken=0.5.2
   - Azure OpenAI API='2024-02-01'
- Streaming of responses in the UX. Answers will begin streaming in chats without having to wait on full answer before returning. 
- Support for adding Service Management Reference and multiple owners on Azure Entry object created for Info Assistant. Following recommended security baselines, these new variables can be used to satisfy policy requirements when deploying. 

## What's Changed
* Update README.md (Typo "Azure Open AI"→"Azure OpenAI") by @hyoshioka0128 in https://github.com/microsoft/PubSec-Info-Assistant/pull/680
* Add MAX_CSV_FILE_SIZE parameter to limit the size of CSV files in Tab… by @dayland in https://github.com/microsoft/PubSec-Info-Assistant/pull/687
* Aparmar/upgrade azure open ai by @ArpitaisAn0maly in https://github.com/microsoft/PubSec-Info-Assistant/pull/682
* Update BICEP references to Terraform in README and documentation files by @dayland in https://github.com/microsoft/PubSec-Info-Assistant/pull/689
* Don't reprocess soft deletes by @georearl in https://github.com/microsoft/PubSec-Info-Assistant/pull/686
* Update Bicep references to Terraform in deployment documentation and … by @georearl in https://github.com/microsoft/PubSec-Info-Assistant/pull/688
* Bump langchain-experimental from 0.0.49 to 0.0.52 in /app/backend by @dependabot in https://github.com/microsoft/PubSec-Info-Assistant/pull/676
* Geearl/7540 storage retention policy by @georearl in https://github.com/microsoft/PubSec-Info-Assistant/pull/691
* Add tags to azurerm_linux_function_app and azurerm_search_service res… by @georearl in https://github.com/microsoft/PubSec-Info-Assistant/pull/692
* Update Azure web app logging configuration in enrichmentapp.tf by @georearl in https://github.com/microsoft/PubSec-Info-Assistant/pull/697
* Update diagnostic setting name in enrichmentapp.tf by @georearl in https://github.com/microsoft/PubSec-Info-Assistant/pull/696
* Geearl/7547 credential lifetime hotfix 3 by @georearl in https://github.com/microsoft/PubSec-Info-Assistant/pull/698
* Update architectural_decisions.md (Typo "Azure Open AI"→"Azure OpenAI") by @hyoshioka0128 in https://github.com/microsoft/PubSec-Info-Assistant/pull/700
* Joshuakr/7432 Chat Streaming by @KronemeyerJoshua in https://github.com/microsoft/PubSec-Info-Assistant/pull/701
* Geearl/upgrade and migrate by @georearl in https://github.com/microsoft/PubSec-Info-Assistant/pull/679
* Fix line spacing issue by @KronemeyerJoshua in https://github.com/microsoft/PubSec-Info-Assistant/pull/709
* Add application insights connection string and key to function app by @dayland in https://github.com/microsoft/PubSec-Info-Assistant/pull/706
* Apply ServiceTreeEntryID via local.env by @georearl in https://github.com/microsoft/PubSec-Info-Assistant/pull/703
* Call is now cancelled when chat mode is switched midstream by @KronemeyerJoshua in https://github.com/microsoft/PubSec-Info-Assistant/pull/711
* updte azurerm_monitor_diagnostic_setting by @georearl in https://github.com/microsoft/PubSec-Info-Assistant/pull/712
* Geearl/7566 clean up by @georearl in https://github.com/microsoft/PubSec-Info-Assistant/pull/708
* cleanup script - added TF state removal by @georearl in https://github.com/microsoft/PubSec-Info-Assistant/pull/716
* Geearl/7564 additional owners by @georearl in https://github.com/microsoft/PubSec-Info-Assistant/pull/718
* chore: Update role assignments for various apps by @georearl in https://github.com/microsoft/PubSec-Info-Assistant/pull/719
* chore: Add check for empty object_id before adding to array by @dayland in https://github.com/microsoft/PubSec-Info-Assistant/pull/720
* update demo question in chat by @wotey in https://github.com/microsoft/PubSec-Info-Assistant/pull/722
* Fixed Generating Answer being displayed on load by @KronemeyerJoshua in https://github.com/microsoft/PubSec-Info-Assistant/pull/725
* Joshuakr/7653 comparison fix by @KronemeyerJoshua in https://github.com/microsoft/PubSec-Info-Assistant/pull/726
* v1.1.1 - Hotfixes for v1.1 by @dayland in https://github.com/microsoft/PubSec-Info-Assistant/pull/724

**Full Changelog**: https://github.com/microsoft/PubSec-Info-Assistant/compare/v1.1...v1.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/PubSec-Info-Assistant/releases/tag/v1.1.1'>v1.1.1</a>.</em>

**Comment by MetteNyland** [comment]
It seems like secure mode is not included in v.1.1.1, is there another time horizon for that?

**Comment by kbvjeppesen** [comment]
Very nice release @dayland and team! Any update on the Secure Deployment feature? We are really looking forward to it 🙌

**Comment by dayland** [comment]
Secure mode will be in Release 1.1.2 that is expected in the next couple of weeks.

**Reply by vishalaryansingh** [reply]
This is an awesome project. I am eagerly waiting for the secure deployment. I tried to implement the VNET integration but encountered issues with API calls to api.cognitive.microsofttranslator.us. I see that there is a PR to resolve this issue.

**Reply by SlavaKeshkov** [reply]
Hi @dayland is there any update on the timeline for secure deployment? Thank you!
